input/text/src/commonMain/kotlin/majestic/InputTextField.kt

```diff
@@ -94,19 +38,15 @@ fun InputTextField(
     onChange: ((String) -> Unit)? = null
 ) {
     val colorBlue = Color(red = 0x00, green = 0x61, blue = 0xFF)
-
     val state = field.state.watchAsState()
-
     var validity: Validity<Any?> by remember(state.output) { mutableStateOf(Valid(state.output)) }
 
     LaunchedEffect(state.output) {
         validity = field.validate()
     }
 
     Column(
-        modifier = Modifier
-            .fillMaxWidth()
-            .padding(vertical = 12.dp)
+        modifier = modifier.fillMaxWidth()
```

Comment:
Since we are passing modifiers, it is better practice to let the user of this composable decide if they need a fillMaxWidth() or they dont. So, do not call it here, but let it be called while using this composable

Reply:
Done